css/donate.css

```diff
@@ -0,0 +1,81 @@
+button {
+  text-align: center;
+  color: white;
+  border: none;
+  background-color: rgb(255, 107, 152);
+  align-items: center;
+  padding: 7px 32px;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+h2 {
+  text-align: center;
+  font-size: 40px;
+}
+
+.donate-story{
+  background-color: rgb(230,230,250);
```

Comment:
the wireframe uses #ECE3F2 for the lighter purple (if u log into the figma, you can click on elements and see exactly what colors are being used)